src/styles/blocks/subscripe.scss

```diff
@@ -0,0 +1,136 @@
+.subscripte {
+  background-image: url(../images/photos/8.png);
+  height: 386px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  @include onTablet {
+    height: 380px;
+  }
+
+  @include onDesktop {
+    height: 550px;
+  }
+
+  &__title {
+    font-family: "Montserrat", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 36px;
+    line-height: 36px;
+    text-align: center;
+    color: #fff;
+
+    @include onTablet {
+      font-size: 48px;
+      line-height: 48px;
+    }
+
+    @include onDesktop {
+      font-size: 48px;
+      line-height: 48px;
+    }
+  }
+
+  &__descr {
```

Comment:
It is recommended to extract font-family, font-style, and font-weight to a common mixin and use it for better maintainability